odoo/models.py

```diff
@@ -3368,34 +3368,31 @@ def write(self, vals):
           triplet is a command to execute on the set of records. Not all
           commands apply in all situations. Possible commands are:
 
-          ``(0, _, values)``
+          ``(0, 0, values)``
               adds a new record created from the provided ``value`` dict.
           ``(1, id, values)``
               updates an existing record of id ``id`` with the values in
               ``values``. Can not be used in :meth:`~.create`.
-          ``(2, id, _)``
+          ``(2, id, 0)``
               removes the record of id ``id`` from the set, then deletes it
               (from the database). Can not be used in :meth:`~.create`.
-          ``(3, id, _)``
+          ``(3, id, 0)``
               removes the record of id ``id`` from the set, but does not
               delete it. Can not be used on
               :class:`~odoo.fields.One2many`. Can not be used in
               :meth:`~.create`.
-          ``(4, id, _)``
+          ``(4, id, 0)``
               adds an existing record of id ``id`` to the set. Can not be
               used on :class:`~odoo.fields.One2many`.
-          ``(5, _, _)``
+          ``(5, 0, 0)``
               removes all records from the set, equivalent to using the
               command ``3`` on every record explicitly. Can not be used on
               :class:`~odoo.fields.One2many`. Can not be used in
               :meth:`~.create`.
-          ``(6, _, ids)``
+          ``(6, 0, ids)``
               replaces all existing records in the set by the ``ids`` list,
               equivalent to using the command ``5`` followed by a command
               ``4`` for each ``id`` in ``ids``.
-
```

Comment:
:+1: good that you make a choice and that devs will stop fucking up the translation function :yum: